Documentation/architecture/net-platform-standard.md

```diff
@@ -134,13 +134,21 @@ Existing PCL projects in VS2013 and VS2015 (excluding UWP targets), can only tar
 
 ### Deprecated monikers
 
-| Platform | NuGet identifier |
-| ---------| --------------- |
-| ASP.NET 5.0 on .NET Framework | aspnet50 |
-| ASP.NET 5.0 on .NET Core | aspnetcore50 |
-| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 |
-| DNX on .NET Core 5.0 | dnxcore50  |
-| Windows 8 | winrt |
+| Platform | Deprecated NuGet identifier | Current NuGet identifier |
+| ---------| --------------------------- | ------------------------ |
+| ASP.NET 5.0 on .NET Framework | aspnet50 | net46 |
+| ASP.NET 5.0 on .NET Core | aspnetcore50 | netstandard1.3 |
+| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 | net451 - net46 |
+| DNX on .NET Core 5.0 | dnxcore50  | dotnet &rarr; dotnet5.4 &rarr; netstandard1.3 |
```

Comment:
> So, what's the correct target framework moniker for library developers whose .NET Portable TFM contains dnxcore50? I'd like to cover this case as well.

Anyone building portable-* can switch to using netstandard.  I believe there is a separate table here that maps the old profiles to the netstandard version to target: https://docs.nuget.org/Create/TargetFrameworks#portable-class-libraries

>netstandardapp1.5 is deprecated too? I thought it lived alongside of netcoreapp1.0?

Yes: deprecated.  It didn't provide any meaningful distinction and it was too easily confused with netstandard.